src/simple_cache_client.rs

```diff
@@ -497,6 +501,181 @@ impl SimpleCacheClient {
         }
     }
 
+    /// Sets dictionary items in a Momento Cache
+    ///
+    /// *NOTE*: This is preview functionality and requires that you contact
+    /// Momento Support to enable these APIs for your cache.
+    ///
+    /// # Arguments
+    ///
+    /// * `cache_name` - name of cache
+    /// * `dictionary_name` - name of the dictionary
+    /// * `dictionary` - hashmap of dictionary key-value pairs
+    /// * `ttl_seconds` - If None is passed, uses the client's default ttl
+    /// * `refresh_ttl`
```

Comment:
I wasn't sure of the exact behavior here. Would be nice to note what true/false mean for this.

Reply:
Great feedback, I can document what this means

Reply:
Addressed in https://github.com/momentohq/client-sdk-rust/pull/76/commits/362fa94c42740de361132cf7371b0a912ca2cdd1